client/client.go

```diff
@@ -275,6 +275,12 @@ func (c *Client) init() error {
 		if err != nil {
 			return fmt.Errorf("failed creating temporary directory for the StateDir: %v", err)
 		}
+
+		p, err = filepath.EvalSymlinks(p)
+		if err != nil {
+			return err
```

Comment:
I believe we should prefix this error with something so we can trace it better (see other errors for examples)